mathml/tools/utils/misc.py

```diff
@@ -1,12 +1,8 @@
-from __future__ import print_function
 import os
 import progressbar
-try:
-    from urllib.request import urlopen
-except ImportError:
-    from urllib2 import urlopen
+from urllib.request import urlopen
 
-UnicodeXMLURL = "https://w3c.github.io/xml-entities/unicode.xml"
+UnicodeXMLURL = "https://github.com/w3c/xml-entities/blob/u14/unicode.xml?raw=true"
```

Comment:
This should have been https://raw.githubusercontent.com/w3c/xml-entities/gh-pages/unicode.xml ; anyway, will update it in the future...